scss/_patterns_side-navigation.scss

```diff
@@ -1,15 +1,145 @@
 @import 'settings';
 
 @mixin vf-p-side-navigation {
+  // STYLING OF SIDE NAVIGATION RESPONSIVE DRAWER
+
+  @keyframes vf-p-side-navigation-expand {
+    0% {
+      transform: translate(-100%);
+    }
+
+    100% {
+      transform: translate(0);
+    }
+  }
+
+  @keyframes vf-p-side-navigation-collapse {
+    0% {
+      transform: translate(0);
+    }
+
+    100% {
+      transform: translate(-100%);
+    }
+  }
+
+  .p-side-navigation__drawer {
+    bottom: 0;
+    left: 0;
+    overflow: auto;
+    position: fixed;
+    top: 0;
+    transform: translateX(-100%);
+    width: 100%;
+    z-index: 12;
+
+    [class*='p-side-navigation'].is-expanded & {
+      @extend %vf-has-box-shadow;
+
+      animation: vf-p-side-navigation-expand map-get($animation-duration, brisk);
+      transform: translateX(0);
+    }
+
+    [class*='p-side-navigation'].is-collapsed & {
```

Comment:
We can't if we want to prevent unexpected animations.

Having 'default' (collapsed) state as one without class means that close animation will trigger on page load for example.

To have animation only triggered by user intent, we need to have both of these classes.